libsession/src/main/java/org/session/libsession/messaging/messages/visible/VisibleMessage.kt

```diff
@@ -101,14 +106,13 @@ data class VisibleMessage(
         proto.applyExpiryMode()
         // Group context
         val storage = MessagingModuleConfiguration.shared.storage
-        if (storage.isClosedGroup(recipient!!)) {
-            try {
-                dataMessage.setGroupContext()
-            } catch (e: Exception) {
-                Log.w(TAG, "Couldn't construct visible message proto from: $this")
-                return null
-            }
```

Comment:
There's something strange with the new logic here. 
`applyExpiryMode` already sets the `expireTimer` and the `expirationType` from the `Message` yet we are now doing more work on `expireTimer` from the recipient's messages?

Reply:
I found that the logic here is actually unnecessary because:
Firstly and most importantly, the disappearing configuration is sent out as a dedicated message. Most of the time these messages should be able to deliver to every participant (contact or groups). 

The code here is trying to attach the expiration settings on **every** outgoing message so to make sure the other end is always aware of the expiration message settings (just in case somehow they haven't received the initial expiration configuration for valid reasons).

After reading the code, I believe the existing `toProto` is sufficient enough to bring the expiration configuration up, there is no need to put in additional expiration settings into the `dataMessage`.

Hence deleting the code here.